waffle/tests/test_management.py

```diff
@@ -329,3 +329,44 @@ def test_delete_some_but_not_all_records(self):
 
         call_command('waffle_delete', flag_names=[flag_1])
         self.assertTrue(get_waffle_flag_model().objects.filter(name=flag_2).exists())
+
+
+class WaffleDeleteUnused(TestCase):
```

Comment:
These tests aren't really testing the desired behavior—deleting flags. Consider updating the command to take a file path of code to search, and passing that in the test calls to confirm items are deleted.